### DISCUSSION TITLE
How to include :: in card without it being treated as a separate card

### DISCUSSION THREAD

**Discussion by pmartin95** [st3v3nmw/obsidian-spaced-repetition#1052] [Q&A] [answered]
I want to create flashcards that use `::`. For example:
`std::boolalpha`::Read/write bool elements as alphabetic strings (`true` and `false`)
However, I don't want to change the default single-line pattern from using :: . Do you know of a way that I can do this?

**Comment by ronzulu** [comment, marked as answer]
Hey there, this seems similar to [Inline flashcard separator conflicts with inline codeblocks #382 ](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/382).

Do you think that covers the issue you are facing?
